src/pages/library/Datasets.test.js

```diff
@@ -35,7 +35,10 @@ describe('Dataset Browser', () => {
   it('shows all datasets in terra', () => {
     expect(
       prepareDatasetsForDisplay(
-        [{ 'dct:title': 'radx-up', 'TerraDCAT_ap:hasDataCollection': [{ 'dct:title': 'RADx-UP' }] }, { 'dct:title': 'not-radx' }],
+        [
+          { 'dct:title': 'catalog-test-included', 'TerraDCAT_ap:hasDataCollection': [{ 'dct:title': 'catalog-test-included' }] },
+          { 'dct:title': 'not-included' },
+        ],
         brands.terra.catalogDataCollectionsToInclude
```

Comment:
Since the catalog itself will be (soon?) removed entirely there's no reason to change this, but as I recall the `catalogDataCollectionsToInclude` feature was added only to support radx and could be removed too.

Reply:
This verifies that we show them all in Terra, which I think we want to keep. If we had one that filtered, I would probably say to remove that, but yeah, the feature could probably get pulled.